std/meta.d

```diff
@@ -169,6 +169,57 @@ unittest
 }
 
 /**
+ * Creates a template, that ignores its parameters and returns a given alias(es).
+ */
+template Const(T...)
+{
+    static if (T.length == 1)
+        alias Const(S...) = T[0];
+    else
+        alias Const(S...) = T;
+}
+
+///
+unittest
+{
+    alias X = Const!int;
+    static assert(is(X!uint == int));
+
+    alias Y = Const!(uint, uint);
+    static assert(is(Y!(int, uint) == AliasSeq!(uint, uint)));
+}
+
+/**
+ * Returns $(D F!(F!(...(F!(X))...))) with n iterations of F.
+ */
+template Iterate(alias F, size_t n, X...)
```

Comment:
This is certainly not quite the iteration but rather application performed  n times.